robosuite/scripts/demo_device_control.py

```diff
@@ -101,6 +101,8 @@
 from robosuite import load_controller_config
 from robosuite.utils.input_utils import input2action
 
+import time
```

Comment:
needed?

Reply:
I don't remember touching this file... this is odd. It might be because further updates happened to v1 from the time I branched off to implement this stuff?

Reply:
fixed